### ISSUE TITLE
Toolbar position problem

### ISSUE THREAD

**Issue by trompx** [Canner/canner-slate-editor#21]
- When selecting multiple lines, the toolbar is centered align while the lines selected are left align.
- When selecting text at the edge of the left side of the screen, the toolbar does not display entirely (overflows outside of screen)

**Comment by chilijung** [comment]
Good catch! Toolbar seems to have many position problems now... FYI, the toolbar implementation is in another repo https://github.com/Canner/slate-toolbar.

Anyway, there are many selection problems still need to fix.

**Comment by chilijung** [comment]
I tried Medium's toolbar, it also has toolbar positioning problem in multiple lines. But haven't dig in the code yet, don't know whether it is possible to have the right position across different blocks.

![kapture 2018-04-19 at 10 35 09](https://user-images.githubusercontent.com/1216029/38968301-6ead29c0-43bd-11e8-88e4-4e0a496db470.gif)
